locales/en/yourloops.json

```diff
@@ -300,6 +302,7 @@
   "join-team-title": "Join the care team {{teamName}}",
   "language": "Language",
   "last-data-update": "Last data update",
+  "last-data-update-tooltip": "Last data Update",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I don't understand the point of this translation key, it's almost the same as the one above and the uppercase "Update" looks weird... What's the reason for it?

Reply:
Discussed with Clément: we need 2 separate translation keys to be able, in some languages, to display an abbreviation of "Last data update" as the title and then the complete sentence in the tooltip. 

:arrow_right: Conclusion: you can keep the translation keys as they are, but the translation of `last-data-update-tooltip` needs to be `"Last data update"` and not `"Last data Update"`.